predict.py

```diff
@@ -33,7 +33,7 @@ def predict_ner(
         job_name, project_id, model_path
     )
     if model_repository == ModelRepository.HuggingFace:
-        from utils.huggingface.predict import huggingface_predict_ner
+        from utils.huggingface.predict_huggingface import huggingface_predict_ner
```

Comment:
is there a reason why this is renamed ?  Sounds like a repetition to me

Reply:
I agree, but mypy does not like predict.py when there is 2 files with the same names